Presentation.Web/app/utility/kendoGridLauncher.ts

```diff
@@ -460,13 +461,20 @@ module Kitos.Utility.KendoGrid {
         originalObject?: any;
     }
 
+    export interface ICustomKendoToolbarDropDownEntry extends IKendoToolbarDropDownEntry {
+        enabled: () => boolean;
+        onClick: () => void;
+        readonly template?: string;
+    }
+
     export interface IKendoToolbarDropDownConfiguration {
         selectedOptionChanged: (selectedOption: IKendoToolbarDropDownEntry) => void;
         availableOptions: IKendoToolbarDropDownEntry[];
     }
 
     export interface IKendoToolbarEntry {
         title: string;
+        getTitle?: () => string;
```

Comment:
I don't assume it is a valid state if both are defined?..
In the builder, make sure to validate that